source/base/Text.ooc

```diff
@@ -174,34 +192,22 @@ Text: cover {
 		this slice(leftPosition, rightPosition - leftPosition + 1)
 	}
 	toString: func -> String {
-		result := String new(this _buffer raw, this count)
-		this free(Owner Receiver)
-		result
+		this _buffer toString()
```

Comment:
I like to put these sort of functions on a single line but I guess it's optional.
`toString: func -> String { this _buffer toString() }`